### ISSUE TITLE
[Bug] Old versions' installers remain in the temporary folder forever

### ISSUE THREAD

**Issue by virtuallyvlad** [standardnotes/forum#466]
On Windows 7, SN seems to keep the installer EXEs in the temp directory (AppData\Roaming) forever, even after they are installed/updated.  As a result, the folder gets perpetually bigger (mine is about 2GB) with each automatic update.

**Comment by mobitar** [comment]
Indeed, we need to clean these up after download. Haven't had a chance to investigate this yet but it's definitely on my radar.

**Comment by rsnitsch** [comment]
Funny. Just today I noticed the same issue. I use WinDirStat from time to time to clean up my drive, and I was stunned to discover that the `AppData\Roaming\Standard Notes` directory takes up 1,8 GB. Of which 1,6 GB are obsolete installer files that have not been cleaned up...

**Comment by mobitar** [comment]
This will be fixed once we update to electron-builder 20.19.1. I'll try this out for the next update, however, electron-builder is such an intricate library that any update is likely not without side effects. https://github.com/electron-userland/electron-builder/issues/2744

**Comment by NelsonMinar** [comment]
I just found a full gigabyte worth of old installers on my system; definitely seems important to clean these up. There's about 80MB each.

**Comment by virtuallyvlad** [comment]
> This will be fixed once we update to electron-builder 20.19.1. I'll try this out for the next update, however, electron-builder is such an intricate library that any update is likely not without side effects. [electron-userland/electron-builder#2744](https://github.com/electron-userland/electron-builder/issues/2744)

On my Win7 setup, these leftover installer EXEs seem to have stopped accumulating after version 3.0.16.  I don't know anything about Electron but I'm guessing [updating it from 1.8 to 5.0](https://github.com/standardnotes/desktop/releases/tag/v3.0.16) also updated electron-builder and electron-updater?

Also, users should be aware that the old installer EXEs weren't deleted by this fix, so **you will still need to go into your temp folder and delete those manually**.
